src/GetTopics.jsx

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+
+export default function GetTopics(props) {
+  const { topics, setTopics } = props;
+  axios.get(`https://nc-news-itve.onrender.com/api/topics`).then((result) => {
+    setTopics(result.data.topics);
```

Comment:
If we're extracting the axios request to a separate file (we should really have all of the axios requests in a separate file at this point), then we shouldn't be setting state in them too. You already have access to the topics/setTopics in Header, so why are they being passed into here?